lib/react_dom_server.dart

```diff
@@ -23,6 +23,13 @@ var renderToStaticMarkup;
 /// Sets configuration based on passed functions.
 ///
 /// Passes arguments to global variables.
+///
+/// > __DEPRECATED.__
+/// >
+/// > This library no longer supports `ReactDomServer` and static markup rendering.
+/// > This will be removed in 6.0.0, along with other configuration setting functions.
+@Deprecated(
+    'This library no longer supports `ReactDomServer` and static markup rendering. Will be removed from this library in the 6.0.0 release.')
```

Comment:
Why are we deprecating this? Was this library, which uses React JS's `ReactDOMServer`, confused with the removed `react_server.dart`?

Reply:
Ah yep, it was! I'll remove that!